ctypesgen/printer_python/printer.py

```diff
@@ -3,17 +3,18 @@
 import sys
 import time
 import shutil
+from os.path import join
 
 from ctypesgen.ctypedescs import CtypesBitfield, CtypesStruct
 from ctypesgen.expressions import ExpressionNode
 from ctypesgen.messages import error_message, status_message
 
 
 THIS_DIR = os.path.dirname(__file__)
-CTYPESGEN_DIR = os.path.join(THIS_DIR, os.path.pardir)
-PREAMBLE_PATH = os.path.join(THIS_DIR, "preamble.py")
-DEFAULTHEADER_PATH = os.path.join(THIS_DIR, "defaultheader.py")
-LIBRARYLOADER_PATH = os.path.join(CTYPESGEN_DIR, "libraryloader.py")
+CTYPESGEN_DIR = join(THIS_DIR, os.path.pardir)
+PREAMBLE_PATH = join(THIS_DIR, "preamble.py")
+DEFAULTHEADER_PATH = join(THIS_DIR, "defaultheader.py")
+LIBRARYLOADER_PATH = join(CTYPESGEN_DIR, "libraryloader.py")
```

Comment:
Why this approach? 
_(is it the pythonic way?
or does it help address any linter warnings about exceeding max chars in a line? 
or something else?)_

- If not,
  let's skip this change for now.

- If yes, 
  as this is existing code (unrelated to this "lean string handling" PR)
  possibly this change belongs in it own separate commit/PR.

Reply:
Path joining is needed rather often, so the `os.path` prefix is a bit tedious.
I fixed up a path joining case below  (`f"{dst}/ctypes_preamble.py"`, which isn't really elegant or windows friendly) and then did this on the way. Explaining or even splitting up all such minor points is too much work.

@TheCodeArtist Please stop comments like this that only take up my time while no one benefits.

Reply:
>  which isn't really elegant or windows friendly
i mention because i have seen `os.path.join` preferred instead of having to lookup what `join()` it is.
(especially as `os.path.join()` handles cross-platform paths properly with the appropriate path-separator)

Also, it would be easier to review/merge PR if such cleanup/useful (but not relevant) changes are kept separate.

> that only take up my time while no one benefits.
You do. i do. ctypesgen does.
In this case reducing all parts not relevant to string cleanup will help shrink this PR.
(which appears to be one of the challenges preventing it from being reviewed/accepted)

Just FYI, in all my comments, please try to have it that i am only trying to help.

Anyways, please carry on with the good work.
Don't mind me too much. 😅